packages/multiselect/src/react/menu.tsx

```diff
@@ -0,0 +1,51 @@
+import { compose, css } from 'glamor'
```

Comment:
Nice organization here.  I like that this is not taken to the nth degree either. That said, I did find index.ts at 340 lines to be a bit of a jumble to read all the way through.  I don't know if you feel similarly. Maybe an opp for further encapsulation of that code.

Reply:
yeah it's pretty large. mostly glueing stuff together...i'd say we keep as is now. splitting out the component would pretty pretty easy but i don't see a huge gain right now